src/docs/product/releases/health/release-details.mdx

```diff
@@ -8,7 +8,7 @@ The Release Details page focuses on an individual release. Elements of the relea
 
 ![View of the release details page showing an overview graph, files changed, and other details.](release_details.png)
 
-Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy, as defined for [Release Health](/product/releases/health/).
+Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy, as defined for [Release Health](/product/releases/health/). Markers will also be displayed here marking adoption stage changes when viewing a time period where that is applicable (and filtering by a single environment).
```

Comment:
Changing this to active rather than passive voice. 

```suggestion
Hover over specific dates and times in the overview graph for a quick perspective of events that crashed, behaved abnormally, errored, or are healthy, as defined for [Release Health](/product/releases/health/). The graph also displays markers showing adoption stage changes when you view a time period where that is applicable (and you've filtered by a single environment).

<Note>

The adoption stage feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>
```